Gir.toml

```diff
@@ -895,6 +894,24 @@ status = "generate"
         const = true
 
 [[object]]
+name = "Gtk.RadioButton"
+status = "generate"
+    [[object.function]]
+    name = "new|new_with_label|new_with_mnemonic"
```

Comment:
This wont work:
- not `name` but `pattern`,
- for patterns always added `^` and `$`, so `|` may not work at all
- due https://github.com/gtk-rs/gir/blob/master/src/analysis/rust_type.rs#L131 lists ignore nullable. I don't remember why @gkoz do that, but maybe empty list works same as null (not checked with radiobuttons)